ckan/public/base/css/main.css

```diff
@@ -2348,7 +2348,8 @@ textarea.form-control-lg {
   line-height: 1.5;
   color: #333333;
   background-color: #fff;
-  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'%3e%3cpath fill='none' stroke='%23343a40' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M2 5l6 6 6-6'/%3e%3c/svg%3e");
+  /* (canada fork only): CSP support */
+  background-image: url("/data/static/img/data_img/chevron-down.svg"), url("/static/img/data_img/chevron-down.svg");
```

Comment:
can we not override this in our site css or does having a data: url anywhere fail the security check?

Reply:
these svg files were added to the ckanext-canada extension so it doesn't feel right referencing them here.

Reply:
uhh you mean actually use CSS properly? yeah haha. Moved to canada plugin. the only thing that would happen if the CSS tried to load a data:image would be that it would be blocked and not rendered on the page. but if the css directives are more specific in the canada plugin, this should be fine